drivers/ptp/ptp_clockmatrix.c

```diff
@@ -33,6 +34,9 @@ MODULE_LICENSE("GPL");
  * over-rides any automatic selection
  */
 static char *firmware;
+
+static struct clk_hw *clock_hw = NULL;
```

Comment:
why not add it to `struct idtcm`?

Reply:
fixed
is there any way to cause idtcm_remove to be called? I saw it is not called on "reboot"

Reply:
you can use rmmod ptp_clockmatrix